2021_05_03.md

```diff
@@ -0,0 +1,123 @@
+# Twitter Space on May 3, 2021
```

Comment:
Wow! From the lost archives! How did you get this recording?

Reply:
You're so welcome!
I was listening and hurriedly taking notes, and halfway through decided I should just turn on a recorder and enjoy listening. The next day I used the recording to take notes and [tweet them](https://twitter.com/csp_frederick/status/1389705338971049986). Then I forgot all about it until Bryan mentioned he couldn't find a copy!

Fortunately I still had the file; unfortunately there's a _lot_ of background noise. Better than nothing I guess.

Reply:
**Much** better than nothing; thank you!